data/mods/Aftershock/mobs/robots.json

```diff
@@ -279,23 +279,23 @@
     "id": "mon_hazmatbot",
     "type": "MONSTER",
     "copy-from": "mon_utilibot",
-    "name": "hazmat bot",
-    "description": "A utility robot designed for cleaning up waste material in hazardous conditions."
+    "name": "TsKBEM Likvidator",
+    "description": "The Likvidator was designed by the TsKBEM design bureau for the Soviet Union as an answer to liquidators post-Chernobyl.  Rather than using humans, easily suspectible to radiation sickness and hard to repair, the Likvidator would do the duties of taking care of toxic hazardous waste instead.  Likvidators are roughly human-sized, bipedal bulky machines; crafted for a utilitarian purpose of cleaning up toxic waste rather than aesthetic."
```

Comment:
```suggestion
    "description": "The Likvidator was designed by the TsKBEM design bureau for the Soviet Union as an answer to the liquidator needs shown by Chernobyl.  Rather than using humans - susceptible to radiation sickness and hard to repair - the Likvidator would instead take care of radioactive, toxic, or otherwise hazardous waste.  Likvidators are roughly human-sized and shaped, but bulky - crafted for the utilitarian purpose of cleaning up toxic waste, not for aesthetics."
```